packages/compiler/server/serverlib.ts

```diff
@@ -465,6 +471,53 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  function getSymbolKindKind(node: Node): { name: string; kind: SymbolKind } | undefined {
+    switch (node.kind) {
+      case SyntaxKind.NamespaceStatement:
+        return { name: node.id.sv, kind: SymbolKind.Namespace };
+      case SyntaxKind.CadlScript:
+        return { name: node.id.sv, kind: SymbolKind.File };
+      case SyntaxKind.EnumStatement:
+        return { name: node.id.sv, kind: SymbolKind.Enum };
+      case SyntaxKind.InterfaceStatement:
+        return { name: node.id.sv, kind: SymbolKind.Interface };
+      case SyntaxKind.OperationStatement:
+        return { name: node.id.sv, kind: SymbolKind.Function };
+      case SyntaxKind.ModelStatement:
+        return { name: node.id.sv, kind: SymbolKind.Struct };
+      case SyntaxKind.ModelProperty && SyntaxKind.StringLiteral:
```

Comment:
don't think that's going to work, didn't even know you could type that in switch expression.

What you want is 
```
case SyntaxKind.ModelProperty: 
  if node.id is a stringliteral:
    return node.id.value
  else 
    return node.id.sv

```